packages/bruno-app/src/components/RawFilePickerEditor/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import path from 'path';
+import { useDispatch } from 'react-redux';
+import { browseFile } from 'providers/ReduxStore/slices/collections/actions';
+import { IconX } from '@tabler/icons';
+import { isWindowsOS } from 'utils/common/platform';
+import slash from 'utils/common/slash';
+
+const RawFilePickerEditor = ({ value, onChange, collection }) => {
+    value = value || '';
+    const dispatch = useDispatch();
+    const separator = isWindowsOS() ? '\\' : '/';
+    const filename = value != '' ? value.split(separator).pop() : value;
+    const title = `- ${filename}`;
+
+    const browse = () => {
+        dispatch(browseFile())
+            .then((filePath) => {
+                const collectionDir = collection.pathname;
+
+                filePath = filePath.startsWith(collectionDir) ? 
+                    path.relative(slash(collectionDir), slash(filePath)) : filePath;
+
+                onChange(filePath);
+            })
+            .catch((error) => {
+                console.error(error);
+            });
+    };
+
+    const clear = () => {
+        onChange(null);
+    };
+
+    return filename.length > 0 ? (
+      <div
+        className="btn btn-secondary px-1"
+        style={{ fontWeight: 400, width: '100%', textOverflow: 'ellipsis', overflowX: 'hidden' }}
```

Comment:
Combining tailwind and inline styles on the same tag could probably become confusing at some point. 
This inline style could also be added to the className as "font-normal w-full text-ellipsis overflow-x-hidden".
The same thing also goes for line 48 where you should be able to just use "w-full"

(Note: I'm not a maintainer, I just thought your PR looked interesting and wanted to contribute 😄 )

Reply:
Thanks for the suggestions. I rarely do front end development so I am sure there are some things that can be approved. It looks like there is a merge conflict now so I'll look into these things when I address that.